ff-netF48-server-sdk.csproj

```diff
@@ -8,6 +8,7 @@
     <Version>1.1.7</Version>
     <PackOnBuild>true</PackOnBuild>
     <PackageVersion>1.1.7</PackageVersion>
+    <AssemblyVersion>1.1.7</AssemblyVersion>
```

Comment:
need this as we can't get access to the package version in the code at run time. We can however generate Assembly info that gets bundled into the DLL, which the executing code can read